python/dgl/homophily.py

```diff
@@ -1,12 +1,25 @@
 """Utils for tacking graph homophily and heterophily"""
 from . import backend as F, function as fn
+from .convert import graph as create_graph
 
-__all__ = ["node_homophily"]
+try:
+    import torch
+except ImportError:
+    pass
+
+__all__ = ["node_homophily", "edge_homophily", "linkx_homophily"]
+
+
+def get_long_edges(graph):
```

Comment:
should internal helper function start with _?

Reply:
I'm fine either way. For this file, it should be clear that only the functions included in `__all__` are external.

Reply:
Sound good.

nit: Maybe rename to get_edges_long, more natural.